keyboards/elephant42/rules.mk

```
@@ -0,0 +1,27 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = caterina
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no      # Enable Bootmagic Lite
```

We highly recommend having this enabled, but isn't strictly required. 
```suggestion
BOOTMAGIC_ENABLE = yes     # Enable Bootmagic Lite
```